src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -150,6 +131,14 @@ const handlers = {
         };
     },
 
+    [options.NEXT_WEEK]: (timezone) => {
+        const nextWeek = now(null, timezone).add(7, "day");
```

Comment:
This maybe better for readability and consistency since you're using 'week' below.
```suggestion
        const nextWeek = now(null, timezone).add(1, "week");
```

Check if this is different from `(7, "days")`